tests/test-bsize-values/test.yaml

```diff
@@ -0,0 +1,31 @@
+args:
+- -k none
+
+requires:
+   min-version: 7.0.0
```

Comment:
Does size not work on previous versions ?

Reply:
I will double check, I thought I had 5 as the min version on this.

Reply:
The 6.x checks fail if I lower the version.

Reply:
Right, `<=` must be min 7 even if bsize < existed before :-)

Reply:
Older versions only supported `>` and `<`

@jmtaylor90 One path is to create a test with just these for 6 and newer and then a test with the other operands for 7 and newer.

Reply:
Sure, I can do that. I will resubmit with the appropriate tests split out.